cognite/client/data_classes/raw.py

```diff
@@ -78,17 +78,30 @@ def __init__(
 
         self._db_name: str | None = None
 
-    def rows(self, key: str | None = None, limit: int | None = None) -> Row | RowList:
+    @overload
+    def rows(self, key: str, limit: int | None = None) -> Row | None:
+        ...
+
+    @overload
+    def rows(self, key: None, limit: int | None = None) -> RowList:
```

Comment:
I believe key should have a default value here, otherwise you're forced to explicitly pass None. We want to allow just omitting it.
```suggestion
    def rows(self, key: None = None, limit: int | None = None) -> RowList:
```

Reply:
Oh yes, nice catch